README.md

```diff
@@ -30,35 +29,30 @@ These environments are accessed through the 2 libraries available to you when yo
 - **CheckoutCardManagerStub**, which powers the Stub environment. Stub allows you to begin testing the APIs in your app, without having to wait for the legal and contractual arrangements required by the other environments, or for backend integration to be completed. The Stub environment is completely isolated; you can provide empty strings instead of tokens in your API calls as no network calls leave the device, and we provide mock data in the responses.
 - **CheckoutCardManager**, which powers the Sandbox and Production environments. Use of these environments requires onboarding with our operations team. During onboarding, you'll receive client credentials, which you will then need to handle on your backend for authentication. You will be expected to manage [Strong Custom Authentication (SCA) requirements](https://www.checkout.com/docs/payments/regulation-support/sca-compliance-guide) as part of accessing the SDK's functionality.
 
-## Migrating from Stub to Sandbox and Production 
+## Migrating from Stub to Sandbox and Production
 
 When you're ready to migrate from the Stub environment, simply update the import statement from `import CheckoutCardManagerStub` to `import CheckoutCardManager`. No additional changes are required, and the public interfaces remain the same.
 
 In the Live version of your app, you'll be expected to provide valid tokens in your requests, which our backend services will serve securely.
 
-***
-# Sample Application
```

Comment:
Why is the reference to Sample Application removed?

Reply:
Not sure why that got removed but have added back in. Thanks for spotting.